tests/tests.py

```diff
@@ -314,6 +314,7 @@ def test_set_and_get_fields(self):
         entry.expires = False
         entry.expiry_time = changed_time
         entry.icon = icons.GLOBE
+        entry.customicon = "9"
```

Comment:
Should this even be valid? I was expecting more something like an actual image file.

Reply:
The way keepass works is that when you set a customicon, it sets the default icon to 0. If you do not want to use the customicon anymore, the CustomIconUUID tag should be removed from the entry. I can only create a separate test for customicons if I modify the test KDBX files; the code will not set customicon to a value which does not actually exist. So setting this to 9 will only result in a customicon with index 9 if at least 10 customicons exist. In all other cases (as it does here) customicon is set to None, which removes the tag from the entry XML.

Reply:
to elaborate, customicons are only stored in the database once and are referred to by entries by using a UUID which is base64 encoded. This PR does not add the possibility to add actual image files as custom icons, but merely allows you to set customicons which already exist in the DB by providing their numerical index.